### PR TITLE
packetbeat 6.0.0

### PR DESCRIPTION
**Pull request by jasontedor**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by ilovezfs — CHANGES_REQUESTED** [review]
This should use a resource block and `setup_install_args` instead of the virtualenv DSL.

**Comment by ilovezfs** [comment]
virtualenv doesn't seem to be used by the build.

**Comment by jasontedor** [comment]
It’s used in `make update`.

**Comment by ilovezfs** [comment]
make update is invoked in auditbeat but not here

**Comment by jasontedor** [comment]
Okay, let me double check that then.

**Comment by jasontedor** [comment]
There was a missing `make update` here, I have updated the PR.

**Comment by jasontedor** [comment]
> This should use a resource block and `setup_install_args` instead of the virtualenv DSL.

I pushed a commit that does this.

**Comment by ilovezfs** [comment]
Same `pip install --no-binary :all:` issue here.

**Comment by jasontedor** [comment]
@ilovezfs I pushed.

File: Formula/packetbeat.rb [hunk 1]
```diff
@@ -15,26 +15,46 @@ class Packetbeat < Formula
...
+      system "python", *Language::Python.setup_install_args(buildpath/"vendor")
+    end
 
-    ENV["GOPATH"] = gopath
+    ENV.prepend_path "PATH", buildpath/"vendor/bin"
 
-    cd gopath/"src/github.com/elastic/beats/packetbeat" do
+    cd "src/github.com/elastic/beats/packetbeat" do
+      # prevent downloading binary wheels
+      inreplace "../libbeat/scripts/Makefile", "pip install", "pip install --no-binary :all"
```
Comment (ilovezfs) [inline Formula/packetbeat.rb]: There needs to be an upstream fix for this in the Makefile with a variable like `PIP_INSTALL_PARAMS` or similar.

Comment (jasontedor) [reply Formula/packetbeat.rb]: I opened elastic/beats#5693. Can we pull this in now (as this will not be available upstream until 6.0.1)?

**Comment by jasontedor** [comment]
@ilovezfs Can we pull this in?

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

File: Formula/packetbeat.rb (hunk 1 above)
Comment (ilovezfs) [reply Formula/packetbeat.rb]: Thanks for taking care of that :)

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks @jasontedor! :ship:'d

**Pull request closed without merging**